cloudgen_firewall/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fec157b4-467e-4faa-8db6-b80baea69b00",
+  "app_id": "cloudgen-firewall",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "CloudGen Firewall is a NGFW that protect networks and Internet traffic",
+    "title": "cloudgen_firewall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Barracuda CloudGen Firewall Overview",
+        "image_url": "images/cloudgen_firewall_overview.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Other Event types of CloudGen Firewall",
+        "image_url": "images/cloudgen_firewall_logs1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Some more Events types of CloudGen Firewall",
+        "image_url": "images/cloudgen_firewall_logs2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Log Collection",
+      "Category::Security",
+      "Offering::Integration",
+      "Queried Data Type::Logs",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id": 48848597,
+      "source_type_name": "cloudgen_firewall",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "cloudgen_firewall",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      }
```

Comment:
```suggestion
```
Does this integration only emit logs? If so, this and the `metadata.csv` can be removed.